src/toil/jobStores/aws/jobStore.py

```diff
@@ -426,7 +426,7 @@ def _importFile(self, otherCls, url, sharedFileName=None, hardlink=False):
             return super(AWSJobStore, self)._importFile(otherCls, url,
                                                         sharedFileName=sharedFileName)
 
-    def _exportFile(self, otherCls, jobStoreFileID, url):
+    def exportFileOrDelegate(self, otherCls, jobStoreFileID, url):
```

Comment:
I think one rename might have been missed.

Reply:
Sorry about that. My search and replace skill needs a serious overhaul. :)
Fixed, and I squashed the commits.